docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -162,7 +163,8 @@ To partially mount an index, you must have one or more nodes with a shared cache
 available. By default, dedicated frozen data tier nodes (nodes with the
 `data_frozen` role and no other data roles) have a shared cache configured using
 the greater of 90% of total disk space and total disk space subtracted a
-headroom of 100GB.
+headroom of 100GB. Shard data for partially-mounted indices is removed from
```

Comment:
May I ask where/how you've confirmed `Shard data for partially-mounted indices is removed from
the node on restart.` ? Else, I'd like to verify this with the team before approving.

Reply:
@kingherc I wrote this based on a support case where we saw frozen nodes going out to the repository to remount the searchable snapshots every reboot, but didn't see the same traffic on cold nodes.  It's been long enough now that I don't have the case number readily available.  I'd love to be proven wrong on this point.

@idegtiarenko One paragraph is referring to fully-mounted snapshot cache and the other is for partially-mounted snapshot cache, and the intention is that there is different behavior and not a contradiction.

Reply:
I see. But I think we need to confirm the mentioned behavior (whether disk cache is kept or not) for both fully-mounted and partially mounted searchable snapshots. My expectation would be that the disk cache re-usage behavior upon restart would be the same, but it's just that partially mounted searchable snapshots probably have evicted much of their data so they may need to re-download a lot.

Since both me and Ievgen are not 100% sure from the top of our heads, some investigation with the team or in the code would be needed to confirm. For snapshots, the ES Distributed Coordination team is responsible, where @idegtiarenko belongs. So @idegtiarenko would you be able to follow up with this PR, e.g., take up reviewing or connect with the Coordination team on it? I'm thinking of removing myself from reviewers (unless I can help due to workload/time of course).